src/banner/index.js

```diff
@@ -1,12 +1,39 @@
 import React from "react";
+import { useNavigate } from "react-router-dom";
 
 const Banner = () => {
   return (
     <>
-      <div className="pt-3">
-        <h1 className="text-align-center">eBuy</h1>
-        <hr style={{ height: "2px", backgroundColor: "#C8C8C8" }}></hr>
+      <div className="pt-2" style={{marginBottom: "-50px"}}>
+          <a href="/">
```

Comment:
idk why but if i hover over certain parts of eBuy it doesn't give me the hyperlink, should we increase padding so if you hover over the general area of eBuy it lets you click?

Reply:
oh its a bug i added a bugs section at the bottom of our doc. I'm trying to get most stuff done fast and then work on bug fixing